src/util.ts

```diff
@@ -1175,6 +1176,28 @@ export class Util {
             range: attr.range
         } as SGAttribute;
     }
+
+    public createCodeAction(obj: CodeActionShorthand) {
```

Comment:
Creating codeActions that actually work are actually kind of hard. I couldn't get the interface items to work, so I had to call `TextEdit.insert`, `TextEdit.replace`, `CodeAction.create`, `URI.file()`, etc. So this function simplifies that process by accepting a more simple structured object. The plugins can return their own `CodeAction` objects if they want, but they might as well just use this function.

Reply:
I like it.